custom_components/healthchecksio/config_flow.py

```
@@ -1,142 +1,220 @@
+                    CONF_SITE_ROOT,
+                    default=user_input.get(CONF_SITE_ROOT, DEFAULT_SITE_ROOT)
+                    if user_input is not None
+                    else DEFAULT_SITE_ROOT,
+                ): str,
+                vol.Optional(
+                    CONF_PING_ENDPOINT,
+                    default=user_input.get(CONF_PING_ENDPOINT, DEFAULT_PING_ENDPOINT)
+                    if user_input is not None
+                    else DEFAULT_PING_ENDPOINT,
+                ): str,
+            }
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)